clients/Android/ultrablue/app/src/main/java/fr/gouv/ssi/ultrablue/database/AppDatabase.kt

```diff
@@ -8,7 +8,7 @@ import androidx.room.RoomDatabase
 /*
```

Comment:
Unrelated to this file, but you forgot to --signoff some of your commits, please rebase to make sure it's present.